test/bench.js

```diff
@@ -73,12 +81,12 @@ function diffBench(imgPath1, imgPath2, diffPath, options, expectedMismatch) {
     const end = performance.now();
     total += end - start;
 
-    diffPNG.data = Buffer.from(new Uint8Array(wasmModule.__getArray(diffPtr)));
+    diffPNG.data = Buffer.from(new Uint8Array(__getArray(diffPtr)));
```

Comment:
This should already return an Array of bytes.

Can't it just be this?

```ts
diffPNG.data = Buffer.from(__getARray(diffPtr));
```

Reply:
It return `Array` not `Buffer`, but we can use `Buffer.from(__getArrayView(diffPtr))`